### PR TITLE
Auto-install core and enterprise plugins during integration tests

### PR DESCRIPTION
**Pull request by bamarni**
https://jira.mesosphere.com/browse/DCOS_OSS-3983

### CODE REVIEW

**Comment by bamarni** [comment]
@mesosphere-ci run integration tests

**Comment by bamarni** [comment]
@mesosphere-ci run integration tests

**Comment by bamarni** [comment]
@mesosphere-ci run integration tests

**Comment by bamarni** [comment]
@mesosphere-ci run integration tests

**Comment by bamarni** [comment]
@mesosphere-ci run integration tests

**Comment by bamarni** [comment]
@mesosphere-ci run integration tests

**Comment by bamarni** [comment]
@mesosphere-ci run integration tests

**Comment by bamarni** [comment]
@mesosphere-ci run integration tests

**Comment by bamarni** [comment]
@mesosphere-ci run integration tests

File: ci/integration-tests.groovy [hunk 1]
```diff
@@ -42,7 +45,8 @@ pipeline {
                 source env/bin/activate; \
                 pip install -r requirements.txt; \
                 flake8 integration; \
-                ./launch_cluster.py ${DCOS_TEST_INSTALLER_URL}"
+                ./launch_cluster.py ${DCOS_TEST_INSTALLER_URL}; \
+                sleep 300"
```
Comment (ArmandGrillet) [inline ci/integration-tests.groovy]: Remove this

**Pull request merged**